src/stacking-group.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from 'react';
+import Avatar from './avatar';
+import Image from './image';
+import * as styles from './stacking-group.css';
+import {useIsInverseVariant} from './theme-variant-context';
+import {vars} from './skins/skin-contract.css';
+import classNames from 'classnames';
+
+type dataArrayType = {
```

Comment:
Types always start with uppercase
Don't include the word Array or Type in a type

You can call it just `type Data =`. But in this case I think it doesn't make sense to create a new type, I'd just inline it in the Props definition